### ISSUE TITLE
replacing real time with pseudotime

### ISSUE THREAD

**Issue by Alexgr97** [broadinstitute/wot#92]
Hey guys, 

Interested to know what your thoughts on this would be:

TL;DR - I'm wondering if it's possible to replace real time with pseudotime. 

I have two stem cell populations at the same day, but with one treated group, leading to a 'more mature' genotype/phenotype compared to the other, despite the same chronological age. 

Each population actually contains ~3 distinct but similar subpopulations of cell types, I'm interested to use a WOT-approach to identify which genes are predictive of a transition from the 'less mature' cell types in the untreated group, to the 'more mature' cell types in the treated group. 

I would likely use another trajectory inference package such as slingshot to identify pseudotimes, I'm wondering what your thoughts are around handling the population size changes, whether I could simply set them to have no overall population size change, and still use WOT. If not, what would you suggest would be the next best thing? At the stage and differentiation state they are at, I would expect there to be little population size change between the two groups anyway. I did consider whether a more appropriate axis of 'population size' might be some genes correlate with 'maturity' and 'immaturity' to replace division and apoptosis, again, interested to know your thoughts on the corollaries of that as an alternative. 

Thanks for your thoughts.

All the best,
Alex
